index.js

```diff
@@ -36,6 +36,11 @@ function getKeys(target) {
 	return Object.keys(target).concat(getEnumerableOwnPropertySymbols(target))
 }
 
+function propertyIsPlain(target, key) {
+	return target[key] === undefined
```

Comment:
Checking for `undefined` isn't technically correct, since it would incorrectly return true in this case:

```js
Object.defineProperty({}, key, {
	configurable: true,
	enumerable: false,
	writable: true,
	value: undefined
})
```

The most correct check I can think of would be `!(key in target)`.